platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/TelemetryImpl.java

```diff
@@ -95,14 +106,13 @@ public boolean setSessionIdRotationInterval(int interval) {
     return telemetry.updateSessionIdRotationInterval(new SessionInterval(interval));
   }
 
+  /**
+   * Set the telemetry api end point base url
+   *
+   * @param baseUrl the end point url for telemetry collection
+   */
   @Override
-  public void onCreateOfflineRegion(@NonNull OfflineRegionDefinition offlineDefinition) {
-    MapEventFactory mapEventFactory = new MapEventFactory();
-    telemetry.push(mapEventFactory.createOfflineDownloadStartEvent(
-      offlineDefinition instanceof OfflineTilePyramidRegionDefinition ? "tileregion" : "shaperegion",
-      offlineDefinition.getMinZoom(),
-      offlineDefinition.getMaxZoom(),
-      offlineDefinition.getStyleURL())
-    );
+  public void setApiBaseUrl(@NonNull String baseUrl) {
+    //telemetry.updateApiBaseUrl(baseUrl);
```

Comment:
Instead of passing base url via api, as far as i can tell, we need to set metadata attribute `com.mapbox.CnEventsServer` to `true` in manifest. that will automatically enable china endpoint.